config.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
from main.py its needed so i can run it with `./main.py` (without need to configure anything more)
from the rest i can remove
it was because the libs were self runnable units before (when they had the __main__ condition)